README.md

```diff
@@ -58,10 +59,39 @@ To [report a vulnerability or bug](https://github.com/stackrox/stackrox/security
 ---
 
 ## Deploying StackRox
-### Installation via Helm
+
+### Quick Installation via Helm
+
+StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+
+**<details><summary>Install StackRox via Helm Installation Script </summary>**
+
+```sh
+/bin/bash -c "$(curl --fail -fsSL https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh)"
+```
+Installing StackRox this way requires multiple CPU resources and may fail if insufficient resources are available. You may use the `--small` option in order to install StackRox on smaller clusters with limited resources.
+```sh
+/bin/bash -c "$(curl --fail -fsSL https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh)" --small
+```
+The script adds the StackRox helm repository, generates an admin password, installs stackrox-central-services, creates an init bundle for shared secrets, and finally creates one secured cluster.
+
+In the folder that the script is run in an init bundle file `stackrox-init-bundle.yaml` is created. Store it safely if you want to create additional secured clusters, otherwise delete it.
+
+Finally, the script will automatically log you into StackRox. A certificate warning may be displayed since the certificate is self-signed, see the next section [First use](#first-use) regarding those warnings. After authenticating you can access the dashboard via
```

Comment:
```suggestion
Finally, the script will automatically log you into StackRox. A certificate warning may be displayed since the certificate is self-signed. See the [First use](#first-use) section to read more about the warnings. After authenticating you can access the dashboard using
```

Reply:
Too long a sentence, I see